libr/anal/esil.c

```diff
@@ -248,7 +248,19 @@ static int internal_esil_mem_read(RAnalEsil *esil, ut64 addr, ut8 *buf, int len)
 			}
 		}
 	}
-	return esil->anal->iob.read_at (esil->anal->iob.io, addr, buf, len);
+	esil->anal->iob.read_at (esil->anal->iob.io, addr, buf, len);
```

Comment:
yes would be, but doing this we break the ESIL until now always return with no error, theses fixes are to can work as previous version and i dont know what hapend if we made the suggest check.